examples/graphql-interfaces-implementing-interfaces/graphql_interfaces_implementing_interfaces.bal

```diff
@@ -0,0 +1,62 @@
+import ballerina/graphql;
+
+service /graphql on new graphql:Listener(4000) {
+
+    // Returning the `Animal` type from a GraphQL resolver will identify it as an interface.
+    resource function get animals() returns Animal[] {
+        return [new Leopard(), new Elephant()];
+    }
+}
+
+// Define the `Animal` interface using a `distinct` `service` object.
+public type Animal distinct service object {
+
+    // Define the `name` field as a resource function definition.
+    resource function get name() returns string;
+};
+
+// Define another `Mammal` interface, which implements the `Animal` interface.
+public type Mammal distinct service object {
+
+    // This denotes that this interface implements the `Animal` interface.
+    *Animal;
+
+    // Add an additional field to the `Mammal` interface.
+    resource function get call() returns string;
+};
+
+// Define the `Leopard` class implementing the `Mammal` interface.
+public distinct service class Leopard {
+
+    // This denotes that this object implements the `Mammal` interface.
+    *Mammal;
+
+    // Since this object implements the `Mammal` interface and the `Mammal` interface implements the
+    // `Animal` interface, this object must implement the fields from the `Animal` interface.
```

Comment:
This seems unclear. Shouldn't it be something like this?

```suggestion
    // `Animal` interface, this object must implement the fields from the both interfaces.
```

Reply:
Fixed with [`2f613bb`](https://github.com/ballerina-platform/ballerina-distribution/pull/3431/commits/2f613bb552c4d09ec049d6ee7af29f035d8104ff)